crates/core/tedge/src/cli/mod.rs

```diff
@@ -98,6 +99,10 @@ pub enum TEdgeOpt {
         relative_links: bool,
     },
 
+    /// Interact with Cumulocity
+    #[clap(subcommand)]
+    C8y(c8y::C8yCmd),
```

Comment:
Wondering if having this concrete enum variant named `C8y` would make it difficult to support profiles, since #3262 is merged and the `c8y@profile` style is preferred over `--profile` arguments.

Reply:
Good point. I will rebase now to assess what's the actual issues.